torch_geometric/nn/conv/gin_conv.py

```diff
@@ -141,11 +142,18 @@ def __init__(self, nn: Callable, eps: float = 0., train_eps: bool = False,
         else:
             self.register_buffer('eps', torch.Tensor([eps]))
         if edge_dim is not None:
-            if hasattr(self.nn[0], 'in_features'):
-                in_channels = self.nn[0].in_features
+            if isinstance(self.nn, torch.nn.Sequential):
+                nn = self.nn[0]
             else:
-                in_channels = self.nn[0].in_channels
+                nn = self.nn
```

Comment:
I think we can safely drop the else here.

Reply:
agree